zerver/lib/actions.py

```diff
@@ -2036,7 +2039,8 @@ def check_message(sender: UserProfile, client: Client, addressee: Addressee,
             return {'message': id}
 
     return {'message': message, 'stream': stream, 'local_id': local_id,
-            'sender_queue_id': sender_queue_id, 'realm': realm}
+            'sender_queue_id': sender_queue_id, 'realm': realm,
+            'widget_content': widget_content}
```

Comment:
I think we might want `check_message` to do some sort of basic validation of `widget_content` (at least data format)